src/web/tools/Vector.ts

```diff
@@ -0,0 +1,54 @@
+import { Directions } from '../../Directions';
+import PointerTracker from './PointerTracker';
+
+export default class Vector {
+  x: number = 0;
+  y: number = 0;
+  unitX: number = 0;
+  unitY: number = 0;
+
+  fromDirection(direction: Directions) {
+    [this.x, this.y] = [this.unitX, this.unitY] = DirectionMappings.get(direction)!;
+
+    return this;
+  }
+
+  fromVelocity(tracker: PointerTracker, pointerId: number) {
+    this.x = tracker.getVelocityX(pointerId);
+    this.y = tracker.getVelocityY(pointerId);
+
+    const minimalVelocity = 1;
```

Comment:
I have mixed feelings about this one. Name is fine and the condition below looks better, but on the other hand I think this type of constants should be declared at least outside of the function (like at the top of the file, or maybe even in `constants.ts` file) cc @j-piasecki.